src/libs/WorkspacesSettingsUtils.ts

```diff
@@ -175,12 +176,22 @@ function getWorkspacesBrickRoads(): Record<string, BrickRoad> {
         }
     });
 
-    Object.values(allReports).forEach((report) => {
+    const actionsByReport: OnyxCollection<ReportActions> = {};
+    Object.keys(reportActions ?? {}).forEach((key) => {
+        if (!reportActions) {
```

Comment:
We should check falsy `key` here right?

Reply:
@eh2077 I guess since report actions can be either `undefined` or `{}` (empty object), would the below condition be ok ?

```ts
if (!reportActions || Object.keys(reportActions ?? {}).length === 0) {
    return;
}
```

Reply:
@ikevin127 Sorry, what I want to point out is that I think we don't need to check 
```js
if (!reportActions) {
    return;
}
``` 
here because we already use nullish coalescing operator `Object.keys(reportActions ?? {})`. What do you think?

Reply:
@eh2077 Oh, I see - cool, I'll remove the check and resolve the other comments in about 30 minutes.

If that's all, you can start working on the checklist and hopefully we can get this ready for final review today.
Thank you!